packages/plugins/src/utils/octav/portfolioElement.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  EvmNetworkIdType,
+  PortfolioElement,
+  PortfolioElementLabel,
+  PortfolioElementType,
+} from '@sonarwatch/portfolio-core';
+import { Cache } from '../../Cache';
+import tokenPriceToAssetToken from '../misc/tokenPriceToAssetToken';
+import { getAmount } from './tokenFactor';
+import BigNumber from 'bignumber.js';
+import { platformId } from '../../plugins/renzo/constants';
+
+export async function createPortfolioElement(
+  tokenAddress: `0x${string}`,
+  balanceValue: bigint,
+  label: PortfolioElementLabel,
+  cache: Cache,
+  networkId: EvmNetworkIdType
+): Promise<PortfolioElement | null> {
+  const tokenPrice = await cache.getTokenPrice(tokenAddress, networkId);
+
+  if (!tokenPrice) return null;
+
+  const balance = getAmount(new BigNumber(balanceValue.toString()));
```

Comment:
```suggestion
  const balance = getAmount(new BigNumber(balanceValue.toString()), tokenPrice);
```
?

https://github.com/Octav-Labs/sonarwatch-portfolio/blob/74f39eb4a04d1b5e9a89e4c4b3abc5e3a2c3165e/packages/plugins/src/utils/octav/tokenFactor.ts#L14-L20 supports receiving an optional `TokenPrice` param.

@noemiesarraillon In order to avoid potential misses like this, we should probably change `getAmount()`'s signature to 
```
tokenPrice: TokenPrice | undefined
```
This will enforce people providing a value and avoid not realizing they skipped the optional param when they could have leveraged if...

Do you agree ?

Reply:
sure